backend/src/services/common/DeploymentFrequency.ts

```diff
@@ -58,7 +59,11 @@ export function calculateDeploymentFrequency(
   const timePeriod = calculateWorkDaysBetween(startTime, endTime);
   const deployFrequencyOfEachPipeline: DeploymentFrequencyModel[] = deployTimes.map(
     (item) => {
-      const passedDeployTimes = item.passed.length;
+      const passedDeployTimes = item.passed.filter(
```

Comment:
This pr is similar as  another pr [[link](https://github.com/thoughtworks/HeartBeat/pull/14)]